test/snippet/range/views/view_all.cpp

```diff
@@ -0,0 +1,16 @@
+#include <string>
+#include <vector>
+
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/range/views/view_all.hpp>       // provides views::all, attention not <seqan3/range/views/all.hpp>!
+
+int main()
+{
+    std::string vec{"foobar"};
+    auto v = vec | seqan3::views::all;   // pipe notation; v is of type std::string_view
```

Comment:
alignment

Reply:
You know, you could have also added suggestions for the alignment fixes :man_shrugging:

Reply:
it was already fun to look through 283 files. I wanted to keep a little bit of fun for you as well.